napari/utils/colormaps/colormap_utils.py

```diff
@@ -39,6 +40,8 @@
     for name, color in zip(primary_color_names, primary_colors)
 }
 
+# dictionay for bop colormap objects
+BOP_COLORMAPS = {k: Colormap(v) for k, v in bopd.items()}
```

Comment:
We need to pass the name of the colormap to the `Colormap` init method. I think this is why the colormap name gets set to `custom` when selecting it in the GUI.

```suggestion
BOP_COLORMAPS = {k: Colormap(v, name=k) for k, v in bopd.items()}
```